<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_15">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Éll hozzáadása: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_add_new_row">
     <property name="text">
      <string>Új sor beszúrása a táblázatba</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </item>
   <item>
    <widget class="QPushButton" name="btn_delete_row">
     <property name="text">
      <string>Kiválasztott sor törlése a táblázatból</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>A kurzort helyezd oda a kódszerkesztőben, ahova beszeretnéd illeszteni az éll hozzáadását.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
